benchmark/net/tcp-raw-c2s.js

```diff
@@ -47,12 +47,12 @@ function main({ dur, len, type }) {
     }, dur * 1000);
 
     clientHandle.onread = function(buffer) {
-      // we're not expecting to ever get an EOF from the client.
+      // We're not expecting to ever get an EOF from the client.
       // just lots of data forever.
       if (!buffer)
         fail('read');
 
-      // don't slice the buffer.  the point of this is to isolate, not
+      // Don't slice the buffer.  the point of this is to isolate, not
```

Comment:
I realize you probably did this with `--fix`, but while we're in here, is it worth it to capitlaize `The` and get rid of the two-spaces separation?

Reply:
Absolutely. I skimmed through the changes quickly to find some of these but I definitely missed a couple. Please keep them coming if you find more :)